commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp

```diff
@@ -123,38 +123,24 @@ long commerceRegionId = BeanParamUtil.getLong(currentCommerceAddress, request, "
 	Liferay.provide(
 		window,
 		'<portlet:namespace />addStreetAddress',
-		function() {
-			var A = AUI();
+		function <portlet:namespace />addStreetAddress() {
```

Comment:
Why name the function here? A name is already being given in the `Liferay.provide` statement -- I think the old way of doing it was right.... unless I'm missing something?

Reply:
Here's more info on `Liferay.provide` in case you're curious: https://issues.liferay.com/browse/LPS-9371.

Reply:
Named functions are useful when debugging
<img width="747" alt="Screenshot 2019-09-26 at 10 24 42" src="https://user-images.githubusercontent.com/1897574/65671712-0523a500-e048-11e9-9a42-aa126fd79e16.png">
<img width="724" alt="Screenshot 2019-09-26 at 10 22 58" src="https://user-images.githubusercontent.com/1897574/65671713-0523a500-e048-11e9-9a36-6d1722eb13ff.png">

Reply:
Alright :). Guess that's a good enough reason for me.